protocols/kad/src/behaviour.rs

```diff
@@ -532,25 +532,25 @@ where
                                 .kbuckets
                                 .bucket(&key)
                                 .map(|b| b.range())
-                                .expect("Not kbucket::Entry::SelfEntry."),
+                                .expect("Not kbucket_priv::Entry::SelfEntry."),
```

Comment:
I think in strings we can retain the old name too.

Reply:
@thomaseizinger Done